### ISSUE TITLE
Is it possible to pull readme file like npmjs

### ISSUE THREAD

**Issue by kunjee17** [fable-compiler/Fable#1841]
Was going through new docs and they are awesome. Now doubt in that. One request or query though, is it possible to pull read me file like npmjs.org is doing ?

I am talking about awesome-fable section. Currently it is going to new windows, it would be great if we can pull the latest read me file from that. If that is possible.

**Comment by alfonsogarciacaro** [comment]
Hi @kunjee17! You mean fetching the README from fable-awesome and integrate them with the documentation every time the docs are built? I guess that's possible but if we want to do that at the end I guess the easiest solution would be to move fable-awesome directly to the docs. If not, every time there's a change in fable-awesome you would have to send a request to rebuild the docs to keep them in sync.

**Comment by kunjee17** [comment]
If possible. I'll go with rebuild option using PR or build trigger. We can use git submodules also right? That way one need to update and push latest code only.

**Comment by MangelMaxime** [comment]
Few people understand `git submodules` so not sure if we want to use them.

And another solution is to use `paket` to fetch the file. We don't have it yet on the `fable-doc` repo but it could also be used to handle the [lightner](https://github.com/fable-compiler/fable-doc/tree/master/lightner) files like grammars, themes, etc. Right now, we are manually copying the file but it's not easy to upgrade them.

**Comment by matthid** [comment]
Another solution is git subtree which is a bit easier to work with than submodule, for a single file however paket might be the better way

**Comment by kunjee17** [comment]
@MangelMaxime I agree with `paket`. I totally forgot that it can pull latest file from `git`. Again it is not urgent but good to have.
